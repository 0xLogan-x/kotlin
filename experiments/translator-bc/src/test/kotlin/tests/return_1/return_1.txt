return_test_1_Int(101) == 101
