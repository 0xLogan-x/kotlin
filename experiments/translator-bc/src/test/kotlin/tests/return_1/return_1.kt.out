@exceptions.str.1 = private unnamed_addr constant  [59 x i8] c"Exception in thread main kotlin.KotlinNullPointerException\00", align 1
declare void @llvm.memcpy.p0i8.p0i8.i64(i8* nocapture, i8* nocapture readonly, i64, i32, i1)
declare i8* @malloc_heap(i32)
declare i32 @printf(i8*, ...)
declare void @abort()
%class.Nothing = type { }
define weak i32 @return_test_1_Int(i32  %return_test_1.x) #0
{
%return_test_1.x.addr = alloca i32, align 4
store i32 %return_test_1.x, i32* %return_test_1.x.addr, align 4
ret i32 101
unreachable
}

